pymobiledevice3/cli/webinspector.py

```diff
@@ -175,7 +174,9 @@ async def inspector_js_loop(inspector: WebinspectorService, app: Application, pa
     while True:
         exp = input('> ')
         try:
-            print(await inspector_session.runtime_evaluate(exp))
+            result = await inspector_session.runtime_evaluate(exp)
+            if result:
```

Comment:
Why?

Reply:
changed to `if exp and not exp.isspace():`, to not print nothing if the the line is empty